tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38
```

Comment:
I've changed this in the master branch to only define environments for the django versions and use the `python_version` matrix in the CI job to test for each Python version. After rebasing use the one in master.